custom-crawls/README.md

```diff
@@ -0,0 +1,42 @@
+# Tutorial: Custom Crawlers
+
+_Note: This tutorial is a work in progress. Please add your feedback to [datatogether/learning](https://github.com/datatogether/learning/issues)!_
+
+## Prerequisites
+
+* You care about a dataset that exists on the web and downloading the data could be benifit from some automation of downloading
```

Comment:
I think we could cut out "You care about a dataset that exists on the web and..."
As the prereq on this is probably more complex (e.g., you have a dataset that can't be downloaded automatically.) 

We may want to revisit the DR guides we prepared and pull in some language: 
https://edgi-govdata-archiving.github.io/guides/